src/forms/input-group.scss

```diff
@@ -18,6 +18,14 @@
   &.inline {
     display: inline-table;
   }
+
+  // within input group, if button exists change focus styles to match input (no offset)
+  &:focus-within {
+    // stylelint-disable-next-line selector-max-type
+    button {
```

Comment:
:focus rather than button to support other types of elements? Not sure if summary is used in this context

Reply:
This is specifically to handle `button` inline with an `input`. I'm not sure `summary` applies here

![image](https://user-images.githubusercontent.com/18661030/142301046-f20bc7a4-0c04-430a-9957-e2d06daf95b6.png)

Reply:
I was thinking about UI like the issue list filter but I see now that it does not use .input-group.
![image](https://user-images.githubusercontent.com/1306747/145740311-0c6c1280-cb6e-4ca6-8d5c-0c851b726e3c.png)